.github/actions/setup-local-volume-provisioner/action.yaml

```diff
@@ -1,37 +1,32 @@
 name: "Setup Local Volume Provisioner"
 description: 'Installs Local Volume Provisioner'
-inputs:
-  repositoryPath:
-    description: "Path to where the project code lives"
-    required: true
-  numberOfPersistentVolumes:
-    description: "Number of persistent volumes that will be provisioned"
-    required: true
 runs:
   using: "composite"
   steps:
-  - name: Prepare host directories
+
+  - name: Checkout Dynamic Local Volume Provisioner
+    uses: actions/checkout@v3
+    with:
+      repository: scylladb/k8s-local-volume-provisioner
```

Comment:
checking out master will break at some point - it will make an update without going trough CI and will bork this repo. Can we copy the deploy files and consumed the image as any other project?

We need to replace the images with sha refs in those files anyways.